scripts/post-pip-compile.sh

```diff
@@ -14,17 +14,6 @@ function show_help {
 function clean_file {
     FILE_PATH=$1
     TEMP_FILE=${FILE_PATH}.tmp
-    # Workaround for https://github.com/jazzband/pip-tools/issues/204 -
-    # change absolute paths for local editable packages back to relative ones
-    FILE_CONTENT=$(<${FILE_PATH})
-    FILE_URL_REGEX="-e (file:///[^"$'\n'"]*)/common/lib/symmath"
-    if [[ "${FILE_CONTENT}" =~ ${FILE_URL_REGEX} ]]; then
-        BASE_FILE_URL=${BASH_REMATCH[1]}
-        sed "s|$BASE_FILE_URL/||" ${FILE_PATH} > ${TEMP_FILE}
-        mv ${TEMP_FILE} ${FILE_PATH}
-        sed "s|$BASE_FILE_URL|.|" ${FILE_PATH} > ${TEMP_FILE}
-        mv ${TEMP_FILE} ${FILE_PATH}
-    fi
```

Comment:
So I was curious why this line specifically calls out `common/lib/symmath`, as there are four other local projects in edx-platform:
* `common/lib/xmodule`
* `common/lib/capa`
* `common/lib/safe_lxml`
* `common/lib/sandbox-packages`
* `openedx/core/lib/xblock_builtin`

all of which suffer from the same type of weird interaction with pip tools that is linked in the comment.

If I'm reading this block of code correctly, it seems like we are only using `common/lib/symmath` to see:
* does `FILE_CONTENTS` contain local Python requirements that need fixing because they contain absolute paths? and
* if so, where on the filesystem is the root of edx-platform? (`BASE_REMATCH[1]` takes on the value of the first regex match in parentheses) 
 

Then, with `BASE_FILE_URL` determined, we fix *all* local requirement paths. I think, rather than deleting this block, it needs to be updated to work without `common/lib/symmath`.

Reply:
I understand now. I was under the impression that this was specific to `symmath`.
Would it be sufficient to replace `symmath` with one of the path's that is still there?
Like `capa`?

Reply:
@Carlos-Muniz Yup, I think that'd be the right move. If I had to recommend one I'd say `xmodule` because that one is the biggest and thus the one most likely to move last.

Reply:
good thinking. I'll keep the code-block, and change it to xmodule.